src/content/development/contribution-guide/_index.en.md

```diff
@@ -290,6 +291,14 @@ Test days will be led by one of the project members, who will be responsible for
 Should a bug be identified during a test day, it should be labeled with an appropriate `testday` label.
 {{% /notice %}}
 
+### Release Notes
```

Comment:
It would make more sense to nest it under "General Availability" as it only pertains to GA releases